lib/schemas/privateThemeConfig.json

```diff
@@ -155,6 +155,29 @@
                 },
                 "uniqueItems": true,
                 "minItems": 0
+              },
+              "translations": {
+                "type": "object",
+                "properties": {
+                  "i18n.description": {
```

Comment:
Same as theme-registry validation, but being pretty strict on this property as we only support `description` atm.